lib/core/src/Cardano/Wallet/DB/Sqlite/TH.hs

```diff
@@ -96,10 +96,11 @@ TxMeta
 -- There is no wallet ID because these values depend only on the transaction,
 -- not the wallet. txInputTableTxId is referred to by TxMeta
 TxIn
-    txInputTableTxId         TxId        sql=tx_id
-    txInputTableOrder        Int         sql=order
-    txInputTableSourceTxId   TxId        sql=source_id
-    txInputTableSourceIndex  Word32      sql=source_index
+    txInputTableTxId           TxId          sql=tx_id
+    txInputTableOrder          Int           sql=order
+    txInputTableSourceTxId     TxId          sql=source_id
+    txInputTableSourceIndex    Word32        sql=source_index
+    txInputTableSourceAmount   W.Coin Maybe  sql=source_amount default=NULL
```

Comment:
Nope, unfortunately. The syntax is a bit weird but that's how we define nullable column with the persistent DSL. This allows for defining a single db schema for both http-bridge and jormungandr.

The difference is actually so small here that I thought this would be better than maintaining two distinct database tables.

Reply:
makes sense